### ISSUE TITLE
Is LTS 19.4.4 Fully compatible with Mysql 8+? Or what is the latest (fully and surely) supported mysql version?

### ISSUE THREAD

**Issue by seansan** [OpenMage/magento-lts#1147]
I read the readme and see some tickets on the subject. But of course need to be absolutely clear on this before we start making some changes ....

Is LTS 19.4.4 Fully compatible with Mysql 8+? 

Or what is the latest (fully and surely) supported mysql version?

Who is on what -modern- version and can you share some experiences? (plz reply)

(also then there is a reference to some experience)

(sorry this is not a bug and will close when we have some feedback in)

**Comment by colinmollenhour** [comment]
Yes, ever since #935/https://github.com/OpenMage/magento-lts/commit/6bd982badeaf6c30429deeea3141332333992225 it is good to go for MySQL 8.0. To my knowledge it should work with any recent version of MariaDb although 10.2 was the latest I tested and any version of Percona should be good as well.
